class-pinterest-for-woocommerce.php

```diff
@@ -712,6 +725,54 @@ public static function maybe_disconnect_advertiser( $old_value, $new_value ) {
 			}
 		}
 
+
+		/**
+		 * Sync the Enhanced Match Support setting with Pinterest.
+		 *
+		 * @param array $old_value The old value of the option.
+		 * @param array $new_value The new value of the option.
+		 */
+		private static function maybe_sync_enhanced_match_support( $old_value, $new_value ) {
+
+			if (
+				! isset( $old_value['enhanced_match_support'] ) ||
+				! isset( $new_value['enhanced_match_support'] )
+			) {
+				return;
+			}
+
+			$tracking_tag = self::get_setting( 'tracking_tag', null );
```

Comment:
Minor: the second parameter here is unnecessary

```suggestion
			$tracking_tag = self::get_setting( 'tracking_tag' );
```